frontend/common/utils/utils.tsx

```diff
@@ -247,15 +247,22 @@ const Utils = Object.assign({}, require('./base/_utils'), {
     }
     return planNames.free
   },
+
   getPlanPermission: (plan: string, permission: string) => {
-    let valid = true
-    const planName = Utils.getPlanName(plan)
-    if (!Utils.getFlagsmithHasFeature('plan_based_access')) {
+    if (
+      !Utils.getFlagsmithHasFeature('plan_based_access') &&
+      global.flagsmithVersion.is_enterprise
```

Comment:
If I'm not mistaken this logic is incorrect, I think we allow features such as audit in open source

Reply:
Thank you @kyle-ssg. @matthewelwell, could you please confirm our business policy regarding the self-hosted free plan? If we have some features enabled for that case, I think we could return true for those in the same block of code.

Reply:
Audit shouldn't be allowed in open source (although it has been to this point). I think therefore this change is correct @kyle-ssg ?

Reply:
Ok, yeah after talking with Ben:

- Self hosted  = same as SaaS without a plan
- EE = same as enterprise plan

I think this means we can remove the plan_based_access flag entirely in this PR.

Reply:
@kyle-ssg @matthewelwell I have a couple of questions in this regard:
In the case of self-hosted (free) AFAIK we should allow unlimited users (invites), while in SaaS free we only allow 1 user. Currently, in the self-hosted free version, we allow unlimited invites for new users, which can be admins or regular users since RBAC is enabled. After these changes, we will only allow admins, since RBAC will be disabled. Is that OK?
If that’s the case should we change this description https://github.com/Flagsmith/self-hosted/blob/ed4a044ddf165e7ec5017cb0dc53e661c3e04b6e/README.md?plain=1#L27 ?
Other behaviors, like limiting the number Projects to only 1, and in general all the non-free plan features that were enabled before for self-hosted free will no longer be available, I guess we are aware of all these changes right?